### PR TITLE
feature/import-account

### PR DESCRIPTION
**Pull request by ThanhNguyen140797**
### Summary 
- Add View mode, Add Public Key screen
- Done Home screen
- Done Add custom token

### Checklist

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [x] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: src/navigation/HomeNavigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            screenOptions={{
+                headerShown: false,
+            }}>
+            {Object.keys(HomeScreens).map((elem: string, index) => (
+                <Stack.Screen
+                    name={elem}
+                    component={HomeScreens[elem].screen}
+                    options={{
+                        title: HomeScreens[elem].title,
+                        // headerShown: elem === AuthenticationRouter.WELCOME ? false : true,
```
Comment (gaupoit) [inline src/navigation/HomeNavigation/index.tsx:21]: Please remove the comment

File: src/navigation/KeyManagerNavigation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            screenOptions={{
+                headerShown: false,
+            }}>
+            {Object.keys(KeyManagerScreens).map((elem: string, index) => (
+                <Stack.Screen
+                    name={elem}
+                    component={KeyManagerScreens[elem].screen}
+                    options={{
+                        title: KeyManagerScreens[elem].title,
+                        // headerShown: elem === AuthenticationRouter.WELCOME ? false : true,
```
Comment (gaupoit) [inline src/navigation/KeyManagerNavigation/index.tsx:21]: Plz remove the comment

File: src/navigation/stack/HomeTabs.tsx [hunk 3]
```diff
@@ -106,8 +101,8 @@ const HomeTabs = () => {
                         tabBarIcon: ({focused}) => (
                             <TabItem {...{focused}} label={'NFT'}>
                                 {
-                                    focused ? <IconMenuNFTActive width={scale(24)} height={scale(24)}/> :
-                                        <IconMenuNFT width={scale(24)} height={scale(24)}/>
+                                    focused ? <IconMenuNFTActive width={scale(25)} height={scale(25)}/> :
```
Comment (gaupoit) [inline src/navigation/stack/HomeTabs.tsx:104]: Please define the constant for the magic number.

File: src/navigation/stack/HomeTabs.tsx [hunk 4]
```diff
@@ -49,6 +49,16 @@ const TabItem = ({focused, children}: TProps) => {
 
 const tabBarHeight = scale(72);
 
+const tabIcon24 = {
```
Comment (ThanhNguyen140797) [inline src/navigation/stack/HomeTabs.tsx:52]: @gaupoit  Is this better?

Comment (gaupoit) [reply src/navigation/stack/HomeTabs.tsx:52]: Yeah, it's more readable now.

File: src/screens/home/HomeScreen/components/TokenComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+const TokenComponent = ({value}: Props) => {
+    return (
+        <CButton>
+            <Row.LR mx={16} style={styles.container}>
+                <Row>
+                    {
+                        value.symbol && (
+                            value.symbol === 'CSPR' ? <Image source={value.icon} style={styles.symbol}/> :
+                                <Image source={{uri: value.icon}} style={styles.symbol}/>
```
Comment (djhahe) [inline src/screens/home/HomeScreen/components/TokenComponent.tsx:20]: could be sorter ?
` <Image source={value.symbol === 'CSPR' ? value.icon : {uri: value.icon}} style={styles.symbol}/>`

File: src/screens/settings/SettingsScreen/index.tsx [hunk 6]
```diff
@@ -1,24 +1,130 @@
...
+        {
+            id: 1,
+            title: 'Lock',
+            icon: () => <IconLock width={scale(32)} height={scale(32)}/>,
+            onPress: () => lockScreen(),
+        },
+    ];
+
+    const openUrl = async () => {
+        const url = 'https://casperdash.io/';
```
Comment (djhahe) [inline src/screens/settings/SettingsScreen/index.tsx:34]: pls move `https://casperdash.io/` to constant file

**Review by djhahe — APPROVED** [review]
LGTM

File: android/app/build.gradle [hunk 7]
```diff
@@ -111,7 +111,8 @@ def enableProguardInReleaseBuilds = false
  * give correct results when using with locales other than en-US.  Note that
  * this variant is about 6MiB larger per architecture than default.
  */
-def jscFlavor = 'org.webkit:android-jsc:+'
+// def jscFlavor = 'org.webkit:android-jsc:+'
+def jscFlavor = 'org.webkit:android-jsc-intl:+'
```
Comment (gaupoit) [inline android/app/build.gradle:115]: Just curios, why do we need to change this config?

Comment (gaupoit) [reply android/app/build.gradle:115]: Please remove the code comment also.

Comment (ThanhNguyen140797) [reply android/app/build.gradle:115]: When i run android, it's has a bug, Intl is undefined
Screen Shot 2022-02-19 at 10 50 22
.

**Review by gaupoit — APPROVED** [review]

**Pull request merged**